hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/HddsUtils.java

```diff
@@ -465,11 +464,11 @@ public static ObjectName registerWithJmxProperties(
   }
 
   /**
-   * Get the current UTC time in milliseconds.
-   * @return the current UTC time in milliseconds.
+   * Get the current time in milliseconds.
+   * @return the current time in milliseconds.
    */
-  public static long getUtcTime() {
-    return Calendar.getInstance(UTC_ZONE).getTimeInMillis();
+  public static long getTime() {
+    return System.currentTimeMillis();
```

Comment:
Thanks @ChenSammi for the analysis and the changes.

It looks like the "current time" change is not related to this, please let me know your thoughts.

Reply:
It's part of another PR.  Will upload a clean patch.